L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryV9Imp2.cc

```diff
@@ -377,9 +377,7 @@ HGCalTriggerGeometryBase::geom_ordered_set HGCalTriggerGeometryV9Imp2::getOrdere
 
 HGCalTriggerGeometryBase::geom_set HGCalTriggerGeometryV9Imp2::getNeighborsFromTriggerCell(
     const unsigned trigger_cell_id) const {
-  HGCalDetId trigger_cell_det_id(trigger_cell_id);
-  geom_set neighbor_detids;
-  return neighbor_detids;
+  throw cms::Exception("FeatureNotImplemented") << "Neighbor search is not implemented in HGCalTriggerGeometryV9Imp2";
```

Comment:
why not?

Reply:
We don't need neighbor information any more in our algorithms. And since it would require a bit of work to deal with neighbors across different wafers, this todo item is not a priority for the moment.